controllers/common.go

```diff
@@ -178,7 +178,8 @@ func (c commonController) finishOAuthExchange(ctx context.Context, r *http.Reque
 
 	code := r.FormValue("code")
 
-	token, err := oauthCfg.Exchange(ctx, code)
```

Comment:
Can you add a comment with an explanation of this change?

Reply:
ok